fdbrpc/AsyncFileNonDurable.actor.h

```diff
@@ -437,6 +446,7 @@ class AsyncFileNonDurable final : public IAsyncFile, public ReferenceCounted<Asy
 			Future<Void> writeEnded = wait(ownFuture);
 			std::vector<Future<Void>> priorModifications =
 			    self->getModificationsAndInsert(offset, length, true, writeEnded);
+			self->minSizeAfterPendingModifications = offset + length;
```

Comment:
Does this need to be maxed with the prior value?

Reply:
Yes, thanks